stable/auth/v1/auth.proto

```diff
@@ -289,6 +289,8 @@ message CheckLoggedInResponse {}
 // TODO: finish
 
 // The service containing authorization/entication methods
+//
+// It is important to note that bots are exempt from AuthService - a persistent authorization token is generated on bot creation.
```

Comment:
It would be good if bots service was mentioned here

Reply:
is it possible to hyperlink it?